platform/platform-api/src/com/intellij/openapi/ui/DialogWrapper.java

```diff
@@ -236,7 +236,7 @@ public void componentResized(ComponentEvent e) {
           if (!myResizeInProgress) {
             myActualSize = myPeer.getSize();
             if (myErrorText != null && myErrorText.isVisible()) {
-              myActualSize.height -= myErrorText.myLabel.getHeight();
+              myActualSize.height -= myErrorText.getMinimumHeight();
```

Comment:
I'd put it myErrorText.getMinimumSize().height without adding new methods.

Reply:
Updated accordingly